dask/dataframe/io/parquet.py

```diff
@@ -165,13 +164,19 @@ def to_parquet(path, df, encoding=default_encoding, compression=None,
         Destination directory for data.  Prepend with protocol like ``s3://``
         or ``hdfs://`` for remote data.
     df : Dask.dataframe
-    encoding : parquet_thrift.Encoding
+    encoding : parquet_thrift.Encoding (future use)
```

Comment:
Correct, it has no effect. Whereas I had wanted to implement things like delta encoding, it's looking increasing further in the future, so this could be removed (here and in fastparquet).